libraries/botbuilder-ai/src/luis-client/model/models.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+/**
+ * NOTE: This class was auto generated by OpenAPI Generator (https://openapi-generator.tech)
+ * and was manually modified to make it compliant with the current implementation of the library.
+ */
+
+export * from './aPIError';
+export * from './compositeChildModel';
+export * from './compositeEntityModel';
+export * from './entityModel';
+export * from './intentModel';
+export * from './luisResult';
+export * from './sentiment';
```

Comment:
Why are these being exported from this file? What's the difference from [`model/index.ts`](https://github.com/microsoft/botbuilder-js/pull/1107/files#diff-0432c52ed06478492a158582768cefb5R13-R19)?

Reply:
There's no difference actually. We are keeping index for exporting and adding a new helper class to handle the ObjectSerializer class and a types file to handle types.